app/assets/stylesheets/application.scss

```diff
@@ -158,3 +158,16 @@ th {
 .alt-header {
     font-family: Pathway Gothic One;
 }
+
+.w-70px {
+    width: 70px;
+}
+
+.main-content {
```

Comment:
why not use the `.offset-*` classes instead of the padding here? from what i can tell in Figma, there's one column of offset on either side.